src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -696,7 +696,7 @@ export const EditSinglelevelTaxonomy: StoryObj<typeof MetadataSidebarRedesign> =
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
 
-        const editButton = await waitFor(() => canvas.getByRole('button', { name: 'Edit My Taxonomy' }));
+        const editButton = await canvas.findByRole('button', { name: 'Edit My Taxonomy' }, { timeout: 30000 });
```

Comment:
is 30s absolutely necessary? `waitFor` is only 1s, have you tried a little longer than 1s?